README.md

```diff
@@ -323,6 +323,9 @@ These variables are the defaults of our roles, if you want to override the prope
 | amqp_protocol | Change the amqp protocol type | amqp |
 | prometheus_url | Prometheus server url | http://host.docker.internal:9090 |
 | web_host | Host where the web instance is listening | http://localhost |
+| web_rpm_listen_port | Port where trento-web will listen on when installed through RPM | 4000 |
```

Comment:
We could share the same web_container_port and wanda_container_port but those actually mean something different inside the container role, there they point to the internal docker port. Let me know what you think.

Reply:
mmm, are you sure they point to the internal port? I don't think so.
I think those 2 ports exactly point to the host port.
Otherwise it doesn't make any sense.
We should use the same variable for both